flexmeasures/ui/templates/crud/account.html

```diff
@@ -400,11 +376,121 @@ <h3>Assets</h3>
 </div>
 {% block paginate_tables_script %} {{ super() }} {% endblock %}
 <script defer>
+  let currentPage = 1;
   const basePath = window.location.origin;
-  const form = document.getElementById("editaccount");
+  const usersApiUrl = basePath + "/api/v3_0/users";
 
+  const form = document.getElementById("editaccount");
   const toastStack = document.getElementById("toast-container");
   const closeToastBtn = document.getElementById("close-all-toasts");
+  const tableBody = document.getElementById("users-table-body");
+  const paginationControls = document.getElementById("pagination-controls");
+
+  async function fetchUsers(page = 1) {
+    try {
+      const response = await fetch(`${usersApiUrl}?page=${page}&per_page=2`);
```

Comment:
Good that you also worked on the account page (though there I also see all users I can possibly see, while it should only be the users of *that* account.

The user API accepts an `account` parameter, so it should be easy.

Reply:
oh, that's true, I forgot about that. I was confused at first but coming back to this comment I get it now, so on the account detail page I will pass that account ID to the API. 👌